.github/workflows/pkg-deps.yaml

```diff
@@ -0,0 +1,102 @@
+name: Package dependencies
+
+on:
+  workflow_call:
+
+jobs:
+  check-dependency:
+    name: Check dependency
+    runs-on: ubuntu-latest
+    if: |
+      github.event_name == 'pull_request' &&
```

Comment:
I am not 100% sure, but I think that approach won't work with workflow_call, as the github.event_name received here will be "workflow_call" and not that of the calling parent "pull_request", so github.base_ref won't be accessible either. The alternative is to use `inputs` under `workflow_call` to pass the branch name.

Reply:
No no. This will work.

> When a reusable workflow is triggered by a caller workflow, the github context is always associated with the caller workflow. The called workflow is automatically granted access to github.token and secrets.GITHUB_TOKEN. For more information about the github context, see "[Contexts](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)."

-- https://docs.github.com/en/actions/using-workflows/reusing-workflows

We are already doing some of this in the `install-slices` workflow.